blockchain/pool.go

```diff
@@ -12,10 +12,10 @@ import (
 )
 
 const (
-	requestIntervalMS         = 250
-	maxTotalRequesters        = 300
-	maxPendingRequests        = maxTotalRequesters
-	maxPendingRequestsPerPeer = 75
+	requestIntervalMS         = 50
+	maxTotalRequesters        = 3000
+	maxPendingRequests        = 2000
+	maxPendingRequestsPerPeer = 2000
```

Comment:
I'm worried this might be too aggressive. Can we get some bandwidth calculations to support what might be reasonable numbers here?  Assuming we are many blocks back and have say 10 peers.

Also, not sure on first glance the implication of `maxPendingRequestsPerPeer` and `maxPendingRequests` being equal

Reply:
Why should the bandwidth matter, other than protocol overhead (data other than the block data itself)? Ideally, the bandwidth won't be the bottleneck and we will use all bandwidth available to us. I would think only our local memory usage is the only resource to worry about (we want CPU used in processing blocks to be the bottleneck).

Reply:
I mean to say that we shouldn't have a situation with excessive memory/routines being used here where the available bandwidth can't keep up